src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -885,6 +882,29 @@
 									Style="{StaticResource RightAlignedToggleSwitchStyle}"
 									ToolTipService.ToolTip="{x:Bind Commands.ToggleShowFileExtensions.HotKeyText, Mode=OneWay}" />
 
+								<!--  Adaptive Layout  -->
```

Comment:
There is a bug in the changes regarding Adaptive layout.
* Manually selecting a layout doesn't turn off the adaptive toggle.
* If I turn off the adaptive toggle in this state, the layout is automatically selected.